src/scales/scale.logarithmic.js

```diff
@@ -138,26 +138,31 @@ module.exports = Scale.extend({
 				var meta = chart.getDatasetMeta(datasetIndex);
 				if (chart.isDatasetVisible(datasetIndex) && IDMatches(meta)) {
 					helpers.each(dataset.data, function(rawValue, index) {
-						var value = +me.getRightValue(rawValue);
+						var value = me._parseValue(rawValue);
 						// invalid, hidden and negative values are ignored
-						if (isNaN(value) || meta.data[index].hidden || value < 0) {
+						if (isNaN(value.min) || isNaN(value.max) || meta.data[index].hidden || value.min < 0 || value.max < 0) {
 							return;
 						}
 
+						if (value.isarr === false) {
```

Comment:
we need this check to keep two logic in consistiency. Scale generates points and labels according to data that passed, some tests on standart logic failed cause with new logic even on general values (not arrays) generated 2 values (2,0) and thus tests failed cause more points where generated . For example if -20 and -50 min|max this will generate -20, -25,-30,-35,-40,-45,-50 for example. But with new logic those code will pass new 0 for scale and we will get 0,-5,-10,-15,-20, -25,-30,-35,-40,-45,-50, so test will fail

Reply:
Oh, okay, I see. `start` would be `0` in the non-array case. That makes sense